### PR TITLE
Adding appRole property to ApplicationCreateParameters and ApplicationUpdateParameters

### PR DESCRIPTION
**Pull request by jillcary**
* Add appRole property to ApplicationCreate and ApplicationUpdate
parameters
* Validated swagger api changes

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/graphrbac/data-plane/stable/1.6/graphrbac.json [hunk 1]
```diff
@@ -2168,6 +2182,13 @@
           },
           "description": "The application permissions."
         },
+        "appRoles": {
+          "type": "array",
+          "items": {
+            "type": "string"
```
Comment (hovsepm) [inline specification/graphrbac/data-plane/stable/1.6/graphrbac.json]: please doublecheck. your other modification declare `"$ref": "#/definitions/appRole"` types as array elements but here you have type string. It is kind of inconsistent.

**Comment by hovsepm** [comment]
@jillcary please fix the linter error - "Error: Could not resolve reference #/definitions/appRole"

**Comment by hovsepm** [comment]
@jillcary please add/modify operation examples that will show valid values for newly added type(s).

**Comment by hovsepm** [comment]
@jillcary your PR introduces a breaking change by adding a new property in the Put(Create) and Get scenarios. The suggested way to do it this kind of changes it to park them till next API version update because it adds a new functionality in the already shipped SDK. SDK users will actually be cleaning out the values that were added previously without actually realizing it. Imagine the following scenario:
1. app using new version of SDK (generated after your changes) creates a record that contains AppRole fields
2. app using old version of SDK (without these changes) gets the record and deserialize all the fields except AppRole because simply it does not know about that field. 
3. the app from n2 now does some modification to other fields and want to update the server record. And as a result of this update it will not provide AppRole values because simply it was thrown away. So AppRole values are just gone.

Please discuss this flow with your team and provide your decision here.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]
@jillcary I would recommend to introduce breaking changes to the next API version.

**Pull request closed without merging**